frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx

```diff
@@ -1,23 +1,30 @@
 import * as React from 'react';
 import { TFunction } from 'i18next';
 import { Shortcut, ShortcutTable } from '@console/shared';
+import { TopologyViewType } from '../../topology-types';
 
 export type Options = {
   supportedFileTypes: string[];
+  isEmptyModel: boolean;
+  viewType: TopologyViewType;
 };
 export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => (
   <ShortcutTable>
-    <Shortcut drag>{t('topology~Move')}</Shortcut>
-    <Shortcut shift drag>
-      {t('topology~Edit Application grouping')}
-    </Shortcut>
-    <Shortcut rightClick>{t('topology~Access context menu')}</Shortcut>
-    <Shortcut click>{t('topology~View details in side panel')}</Shortcut>
-    <Shortcut hover>{t('topology~Access create connector handle')}</Shortcut>
+    {!options.isEmptyModel && options.viewType === TopologyViewType.graph && (
+      <>
+        <Shortcut drag>{t('topology~Move')}</Shortcut>
```

Comment:
You should be `data-test-id`'ing these items and checking if they are there. You're data-test ids for lower items is fine, but you should be testing for the whole item not just a bunch of loose operations. Checking to see if something has `drag` is _okay_ but you really should be writing a test to see if `Move` is there when the condition is met.

Reply:
Done